dev-cpp/jwt-cpp/jwt-cpp-0.6.0.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Thalhammer/jwt-cpp.git"
+else
+	SRC_URI="https://github.com/Thalhammer/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="header only library for creating and validating JSON Web Tokens in C++11"
+HOMEPAGE="https://thalhammer.github.io/jwt-cpp/"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="doc +picojson test"
+
+DEPEND="${RDEPEND}
+	dev-cpp/nlohmann_json
+	dev-libs/openssl
```

Comment:
You're not defining RDEPEND.

Also, if we're linking against openssl, :=.

Reply:
Thanks for spotting, this is fixed now. In case of `jwt-cpp`, which is headers-only, we don't link, but it's very true for `scitokens-cpp`.